Source/WebCore/page/QuirksData.h

```diff
@@ -28,6 +28,7 @@
 namespace WebCore {
 
 struct WEBCORE_EXPORT QuirksData {
+    bool inputMethodUsesCorrectKeyEventOrder : 1 { false };
```

Comment:
Nit - it looks like this first section is for `is(Domain/Website)` flags. Maybe this could be moved to just before `bool hasBrokenEncryptedMediaAPISupportQuirk`?